__tests__/integration.js

```diff
@@ -57,4 +100,28 @@ describe("tests jest snapshots of executed notebooks", () => {
     const nb = JSON.parse(fs.readFileSync(nbPath));
     expect(nb).toMatchSnapshot();
   });
+
+  it("tests new-code-cells", () => {
+    const nbPath = "notebooks/new-code-cell.ipynb";
+    const nb = JSON.parse(fs.readFileSync(nbPath));
+    expect(nb).toMatchSnapshot();
+  });
+
+  it("tests new-text-cells", () => {
+    const nbPath = "notebooks/new-text-cell.ipynb";
+    const nb = JSON.parse(fs.readFileSync(nbPath));
+    expect(nb).toMatchSnapshot();
+  });
+
+  it("tests change theme to dark", () => {
+    const nbPath = "notebooks/dark-theme.ipynb";
+    const nb = JSON.parse(fs.readFileSync(nbPath));
+    expect(nb).toMatchSnapshot();
```

Comment:
Checking the notebook snapshot might be silly with theme changing, since the theme is just a css variable swap on the frontend.

Reply:
That's true. The scenario I had in my head is that it might catch some unintentional action triggered by changing theme, but it would only catch one that actually changed the notebook which is probably pretty unlikely. Will remove 👍

Reply:
When we get console errors, do they end up in logs somewhere for us to see?

Reply:
That's a great question. I don't think there is by default, but I think we could do something like `--json --outputFile test-logs.json` and store it as an artifact (at least to catch and keep the test logs). I don't actually know if that would capture console errors, though 🤔. I think I need to come up with a simple way to trigger a console error to test this out on my end.